browser/brave_wallet/brave_wallet_provider_impl_unittest.cc

```diff
@@ -925,6 +938,8 @@ TEST_F(BraveWalletProviderImplUnitTest, AddAndApproveTransaction) {
   EXPECT_EQ(infos[0]->tx_status, mojom::TransactionStatus::Unapproved);
   EXPECT_EQ(infos[0]->tx_hash, tx_hash);
 
+  EXPECT_EQ(*GetTransactionInfo(infos[0]->id), *infos[0]);
```

Comment:
Please also test the case that transaction is not found here.

Reply:
Done